src/components/bubble/_bubble.scss

```diff
@@ -8,48 +8,54 @@ $borderSize: 3px;
 $triangleSize: 12px;
 $innerTriangleSize: $triangleSize - 2 * $borderSize + 1;
 
-.mint-bubble {
-  @include component;
-  overflow: visible;
-  background: $bubbleBackground;
-  border: $borderSize solid $bubbleBorderColor;
-  border-radius: $bubbleBorderRadius;
-  padding: 15px;
-  box-shadow: $bubbleBoxShadow;
-
-  &:before {
-    content: "";
-    border: $triangleSize solid transparent;
-    border-bottom-color: $bubbleBorderColor;
-    position: absolute;
-    top: -(2 * $triangleSize);
-    right: 0;
-    left: 0;
-    width: 0;
-    margin: 0 auto;
-  }
+$include-html-classes: true !default;
```

Comment:
we were thinking about that but we decided to make each file includable separately and do to that this variable has to be defined in each file.
@mr-mig wdyt?